tests/cli/test_cli_classification_mol.py

```diff
@@ -148,3 +149,4 @@ def test_train_outputs(monkeypatch, data_path, tmp_path):
     checkpoint_path = tmp_path / "model_0" / "checkpoints" / "last.ckpt"
 
     model = MPNN.load_from_checkpoint(checkpoint_path)
+    assert model is not None
```

Comment:
This is actually just to satisfy a `flake8` error. `model` would otherwise be an unused variable, so I put this in to satisfy that. The test could probably do something more involved? But I think this is fine.

Reply:
I think a better solution is to change it to not assign the output to a variable `MPNN.load_from_checkpoint(checkpoint_path)`. All the test is really doing is checking that we can load the model without errors, so we don't need to catch the output. But if we should add more tests and make it more involved, then it is probably better to leave `assert model is not None` as a place holder to remind us to add those tests. Your call.